src/pubsub/ua_pubsub_security.h

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2017-2018 Fraunhofer IOSB (Author: Jan Hermes)
+ */
+
+#ifndef UA_SERVER_PUBSUB_SECURITY_H
+#define UA_SERVER_PUBSUB_SECURITY_H
+
+#include <open62541/plugin/log.h>
+#include <open62541/types.h>
+#include <open62541/types_generated.h>
+
+#include "ua_pubsub_networkmessage.h"
+
+UA_StatusCode
+verifyAndDecrypt(const UA_Logger *logger,
```

Comment:
A header file for a single function, that's a bit too fine-grained.
Merge this into the general pubsub.h header.